[AOTI] Handle inplace output in ProxyExecutor

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #136534
* __->__ #137660

Summary: https://github.com/pytorch/pytorch/pull/137401 didn't fix the underlying inplace output issue.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @chauhang

desertfire committed:
[AOTI] Handle inplace output in ProxyExecutor

Summary: https://github.com/pytorch/pytorch/pull/137401 didn't fix the underlying inplace output issue.

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Handle inplace output in ProxyExecutor"


Summary: https://github.com/pytorch/pytorch/pull/137401 didn't fix the underlying inplace output issue.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Handle inplace output in ProxyExecutor"


Summary: https://github.com/pytorch/pytorch/pull/137401 didn't fix the underlying inplace output issue.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

chenyang78 reviewed (commented):
Failing test seems to be real.

desertfire committed:
Update on "[AOTI] Handle inplace output in ProxyExecutor"


Summary: https://github.com/pytorch/pytorch/pull/137401 didn't fix the underlying inplace output issue.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Handle inplace output in ProxyExecutor"


Summary: https://github.com/pytorch/pytorch/pull/137401 didn't fix the underlying inplace output issue.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge

desertfire commented:
@pytorchbot merge -f "test_cuda_load is flaky"

desertfire commented:
@pytorchbot revert -m "Fails in fbcode" -c nosignal

desertfire committed:
Update on "[AOTI] Handle inplace output in ProxyExecutor"


Summary: https://github.com/pytorch/pytorch/pull/137401 didn't fix the underlying inplace output issue.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]
